tests/unity/CMakeLists.txt

```diff
@@ -19,15 +20,19 @@ macro(configure_unity_conf_file mock_name exclude_list out_config_file)
                    ${out_dir}/${unity_config_file_name}
     )
     set(${out_config_file} ${out_dir}/${unity_config_file_name})
-  elseif("${exclude_list}" STREQUAL "")
+  elseif("${exclude_fn_list}" STREQUAL "" AND "${exclude_word_list}" STREQUAL "")
     # Just return the common file.
     set(${out_config_file} ${out_dir}/${unity_config_file_name})
   else()
     set(CMOCK_STRIPPABLES ":strippables:\n")
-    foreach(ex ${exclude_list})
+    foreach(ex ${exclude_fn_list})
       string(CONFIGURE "        - '(?:${ex}\\s*\\(+.*?\\)+)'\n" strip_regex)
       set(CMOCK_STRIPPABLES "${CMOCK_STRIPPABLES}${strip_regex}")
     endforeach()
+    foreach(ex ${exclude_word_list})
+      string(CONFIGURE "        - '${ex}'\n" strip_word)
```

Comment:
curious, don't you still need `(?:${ex})`, so that users can simply call the function as `WORD_EXCLUDE foo`, and not having to call as `WORD_EXCLUDE "(?:foo)"`.

That was the original intention of the function, that users didn't have to consider the needed use of `(?:  )` when using CMock, they just specify what they want to exclude.

Or have I missed something wrt `WORD_EXCLUDE` ?

Reply:
It seems that the two things are basically equivalent, from the doc I could find about it:
`The (?:…) construct provides grouping without capturing. That is, it combines the terms it contains into an atomic whole without creating a backreference. This benefits performance at the slight expense of readability.`

Nevertheless, I have updated it as you suggested; both work fine for me.